app/src/docs/components/CanvasModuleHelp/index.jsx

```diff
@@ -79,6 +81,11 @@ type Props = {
     className?: string,
 }
 
+const ModuleBusStop = styled(BusStop)`
+    position: relative;
+    top: -48px;
```

Comment:
I make the scroller go a bit higher so we don't touch the edge of the module name (the text).